gfx/libresprite/build

```
@@ -0,0 +1,23 @@
+#!/bin/sh -e
+
+sed -i '/pthread_detach/d' src/base/thread.cpp
+
+git submodule update --init --recursive
+
+export DESTDIR="$1"
+
+mkdir build && cd build
+cmake \
+  -DUSE_SHARED_CURL=ON \
+  -DUSE_SHARED_ZLIB=ON \
+  -DUSE_SHARED_JPEGLIB=ON \
+  -DUSE_SHARED_LIBPNG=ON \
+  -DUSE_SHARED_LIBWEBP=ON \
+  -DUSE_SHARED_PIXMAN=ON \
+  -DUSE_SHARED_FREETYPE=ON \
+  -DUSE_SHARED_GIFLIB=ON \
+  -DUSE_SHARED_GTEST=ON \
```

```suggestion
  -DUSE_SHARED_GTEST=OFF
```
With this we don't need `googletest`, please remove them.